plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/diagram/benchmark/DummyCreationTool.java

```diff
@@ -1,45 +0,0 @@
-/*******************************************************************************
```

Comment:
Why this file is removed ? Probably deprecated, but I see no link with the subject...